packages/components/src/Components/DateFormat/helper.js

```diff
@@ -16,9 +19,25 @@ const relativeTimeTable = [
 
 const exact = (value) => value.toUTCString().split(',')[1].slice(0, -4).trim();
 
+export const addTooltip = (date, element) => (
+    <Tooltip
+        content={
+            <div>date</div>
+        }
```

Comment:
This should be wrapped in `{}`, also no need to space it that much. `content={ <div>{date}</div> }`